CONFIG.md

```diff
@@ -374,12 +373,11 @@ This is an example of the settings for the validate tasks:
   }
   "phplint": {
     "dir": [
-      "<%= config.srcPaths.drupal %>/**/*.php",
```

Comment:
I think we can at minimum include linting for template.php, and perhaps the best-practice naming convention for a templates and includes directory. Open to suggestions, but it would be something like:

`'<%= config.srcPaths.drupal %>/themes/' + key + '/template.php',`
`'<%= config.srcPaths.drupal %>/themes/' + key + '/templates/**/*.php',`
`'<%= config.srcPaths.drupal %>/themes/' + key + '/includes/**/*.{inc,php}'`